easybuild/easyconfigs/b/BuildEnv/BuildEnv-1.0-intel-2015a-default.eb

```diff
@@ -0,0 +1,13 @@
+easyblock = "BuildEnv"
```

Comment:
same as 'software' name, so no need to include this explicitly

but maybe `name` should be something like `'eb-build-env'`?